server/jamovi/server/compute/functions.py

```diff
@@ -244,6 +244,17 @@ def VMED(values: float):
     return stats.median(values)
 
 
+@column_wise
+def VMAD(values: float):
+    median = VMED(values)
+    return VMED([abs(value - median) for value in values])
```

Comment:
sorry for the delay on this,

i think the use of a "list comprehension" loads the entire column into a list in memory, where as i think we can use a `map()` instead.

i.e.

```
return VMED(map(lambda v: abs(v - median), values))
```

Reply:
Done!
Your suggestions are always precious pearls.
TA